packer/ansible/playbook.yml

```diff
@@ -12,6 +12,14 @@
     - banner
     - persist_journald
 
+# Any instances that are built on Debian Buster should have Python 2 removed
```

Comment:
It's probably worth considering moving the upgrade and python steps to separate playbooks as is done [in cisagov/skeleton-docker](https://github.com/cisagov/skeleton-packer/blob/develop/src/packer.json#L65-L79).  The advantage is that Ansible "resets" and re-determines what version of python to use at the beginning of each provisioning step.  This way all the foo you really care about is installed under Ansible via python 3.

It probably doesn't matter much for the bastion, but it will matter a great deal for other AMIs that use the `pip` Ansible module.

Reply:
Ah, I missed that.  Thanks for pointing it out.